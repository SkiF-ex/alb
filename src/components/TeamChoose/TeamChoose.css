.coose_block {
    display: flex;
    flex-direction: column;
    align-items: center;
    height: 100%;
    justify-content: center;
    font-size: 50px;
    font-style: italic;
    color: rgb(20, 118, 184);
}

.coose_block p {
    margin: 0;
}

.choose_button {
    background-color: transparent;
    border: 1px solid black;
    font-size: 30px;
    margin-top: 50px;
}