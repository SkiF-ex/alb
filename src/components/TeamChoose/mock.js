export const DATA_MOCK_WEB_TEAM = {
  title: 'WEB TEAM',
  subtitle: 'BLAZING FAST WEB',
  description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.`,
  subdescription: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.`
}

export const DATA_MOCK_MOBILE_TEAM = {
  title: 'WEB MOBILE TEAM',
  subtitle: 'BLAZING FAST WEB',
  description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.`,
  subdescription: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.`
}
