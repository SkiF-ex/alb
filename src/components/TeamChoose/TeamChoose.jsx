import React from "react";
import { Link } from "react-router-dom";

import './TeamChoose.css';

const TeamChoose = () => {
    return (
        <>
            <Link to="/" className="button_item button_previous">
                <button className="button_item"/>
            </Link>
            <section id="webTeam">
                <div className="page">
                    <div className="coose_block">
                        <p>WEB TEAM</p>
                        <p>ALBUM</p>
                        <Link to="/our_team:web">
                          <button className="choose_button web">Open</button>
                        </Link>
                    </div>
                </div>
                <div className="page">
                    <div className="coose_block">
                        <p>MOBILE TEAM</p>
                        <p>ALBUM</p>
                        <Link to="/our_team:mobile">
                          <button className="choose_button mobile" >Open</button>
                        </Link>
                    </div>
                </div>
            </section>
            <button className="button_hidden button_item" />
        </>
    );
}

export default TeamChoose;
