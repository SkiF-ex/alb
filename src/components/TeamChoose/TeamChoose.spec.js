import React from "react";
import { MemoryRouter } from "react-router-dom";
import TeamChoose from "./TeamChoose";
import { act, create } from 'react-test-renderer';

describe('TeamChoose component', () => {

  it('should render component', () => {
    let component;

    act(() => {
      component = create(
        <MemoryRouter>
          <TeamChoose />
        </ MemoryRouter>
      );
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
})
