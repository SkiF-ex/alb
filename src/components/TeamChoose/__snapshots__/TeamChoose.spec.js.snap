// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamChoose component should render component 1`] = `
Array [
  <a
    className="button_item button_previous"
    href="/"
    onClick={[Function]}
  >
    <button
      className="button_item"
    />
  </a>,
  <section
    id="webTeam"
  >
    <div
      className="page"
    >
      <div
        className="coose_block"
      >
        <p>
          WEB TEAM
        </p>
        <p>
          ALBUM
        </p>
        <a
          href="/our_team:web"
          onClick={[Function]}
        >
          <button
            className="choose_button web"
          >
            Open
          </button>
        </a>
      </div>
    </div>
    <div
      className="page"
    >
      <div
        className="coose_block"
      >
        <p>
          MOBILE TEAM
        </p>
        <p>
          ALBUM
        </p>
        <a
          href="/our_team:mobile"
          onClick={[Function]}
        >
          <button
            className="choose_button mobile"
          >
            Open
          </button>
        </a>
      </div>
    </div>
  </section>,
  <button
    className="button_hidden button_item"
  />,
]
`;
